packages/browser/src/Component/Widget/Server/Documents/View/DocumentInspectorWindow.js

```diff
@@ -25,7 +25,14 @@ export class DocumentInspectorWindow extends AbstractDocumentWindow {
      * Represents a list of extensions. An extension can either be a button or
      * a panel.
      *
-     * @type {Object<string, DocumentInspectorExtension>}
+     * @type {Object<string, {
```

Comment:
why not using DocumentInspectorExtension type define above in the document ?
this type is only used to type this.extensions
if it doesnt match the type of extension just modify DocumentInspectorExtension or remove it

Reply:
fixed